### PR TITLE
[Added] Escape the obstacle game Using Hooman Library

### PR DESCRIPTION
**Pull request by prateek10201**
**Description**
Navigate the _rectangle_ block through falling _spheres_ using arrow keys. Dodge spheres to continue playing, with the game getting faster as you play, the speed of spheres increases. 

**Instructions:**
- Use left and right arrows to navigate. 

**Files Added**
- **hooman/demos/escape_the_obstacles.py**

### CODE REVIEW

**Review by Abdur-rahmaanJ — CHANGES_REQUESTED** [review]
@prateek10201 the file is empty ... maybe you forgot to save?

**Comment by prateek10201** [comment]
@Abdur-rahmaanJ , I have made changes, could you please check?

File: hooman/demos/escape_the_obstacles.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
+import random
+import pygame
+from hooman import Hooman
+
+# Initialize pygame and hooman
+pygame.init()
```
Comment (Abdur-rahmaanJ) [inline hooman/demos/escape_the_obstacles.py:6]: We don't need this line in Hooman

File: hooman/demos/escape_the_obstacles.py [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    # Update the display
+    hapi.flip_display()
+    score += 1  # Increase score
+    hapi.text(f"Score: {score}", 10, 10)
+
+    # Limit the frame rate
+    clock.tick(fps)
+
+# Quit the game
+pygame.quit()
```
Comment (Abdur-rahmaanJ) [inline hooman/demos/escape_the_obstacles.py:90]: There is no need to add this line also.

**Review by Abdur-rahmaanJ — CHANGES_REQUESTED** [review]

File: hooman/demos/escape_the_obstacles.py (hunk 1 above)
Comment (prateek10201) [reply hooman/demos/escape_the_obstacles.py:6]: @Abdur-rahmaanJ , you are specifying pygame.init() right? I will remove it and commit again.

Comment (Abdur-rahmaanJ) [reply hooman/demos/escape_the_obstacles.py:6]: Yes!

**Comment by prateek10201** [comment]
I have made changes, could you please review now? @Abdur-rahmaanJ

**Review by Abdur-rahmaanJ — APPROVED** [review]

**Pull request merged**